Public/Src/Cache/ContentStore/Library/Stores/QuotaManagement/QuotaKeeper.cs

```diff
@@ -34,37 +33,38 @@ public enum QuotaKeeperCounters
     /// </summary>
     public abstract class QuotaKeeper : StartupShutdownBase
     {
+        /// <nodoc />
+        protected readonly bool PurgeAtStartup;

```

Comment:
>PurgeAtStartup [](start = 32, length = 14)

Are this changes intended in this PR, They look like the same changes from a differnt PR...

Reply:
Removed them. Thanks.